p44/src/hashmap_macro.rs

```diff
@@ -0,0 +1,16 @@
+// In lib.rs create a public hash_map! macro which allows to create HashMap like this:
+// let map = hash_map!(
+// 42 => true, 64 => false, 128 => true,
+// );
+// Write an integration test fo the macro
+
+#[macro_export]
+// Source: src/macros.rs - vec!
+macro_rules! hashmap {
+    () => {
+        HashMap::<u64, bool>::new();
+    };
+    ($($key:expr => $value:expr $(,)?)+) => {
+        HashMap::<u64, bool>::from([$(($key, $value) ,)*]);
```

Comment:
This macro should work with any types of keys and values. Plus, you unnecessarily create an array here. Try to write code using the `isnert` method.

Reply:
1. types: unsure
2. array - done